packages/create-resolve-app/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "create-resolve-app",
+  "version": "0.0.12",
+  "description": "This package generates boilerplate web application based on resolve framework",
+  "bin": {
+    "create-resolve-app": "./dist/bin/index.js"
```

Comment:
Where is this file? If supposed `/src/index.js`, that it should contant shebang construction in first line